### ISSUE TITLE
[Bug]: Items stored inside Immersive Engineering tools disappear

### ISSUE THREAD

**Issue by TheInfamousAlk** [AllTheMods/ATM-6#2694]
### Possible Fixes

Yes

### Modpack Version

1.8.9

### What happened?

I have a modded server that I host. There have been multiple instances of items disappearing from inside immersive engineering tools.

The first instance was an engineer's toolbox wiping all items stored inside. I crafted the toolbox, used it for a day (real time), logged off, and the next day I logged on and its contents were empty. I had several items from several mods inside. Not just IE stuff.

The 2nd instance was when I crafted a mining drill and buzz saw (again, since they were wiped) and upgraded them with blades, drill head, and several upgrades and shaders. It worked, and it stored the upgrades just fine. Immediately after, I was just going around killing mobs with my new railgun, when I open my inventory and find that the drill and buzzsaw now have no blade, drill head, or any upgrades. The shaders were the only thing that was not wiped. I did not log off, change game mode from survival, or do anything strange like teleport. I simply crafted them, installed the upgrades, and started walking around killing mobs nearby. (in the log I pasted, the disappearance happened maybe 10-15 minutes before the end of the log, which is when I copied it)

In other words, the NBT data is getting wiped/reset for these tools, as that is where the storage is. I do not know how this is happening. 

I have read this happens when you spawn in these items, however I crafted them in survival with an ME crafting interface. 

### latest.log

https://gist.github.com/TheInfamousAlk/1c9c45af039687a42b921a3f90c76214

**Comment by TheInfamousAlk** [comment]
I have an update. It appears the NBT data is still there temporarily. When I hold tab I can see the NBT data, and it shows the upgrades in there. They however, are not there and do not work. When I put the tool (drill or buzzsaw) into the engineer's workbench while in this state, the upgrades do not show up. I can still see the upgrades in the NBT data while it is in the workbench. But when I click one of the empty upgrade slots, all of the NBT data is wiped. I do not know if this is helpful or not. 

In other words, the tool is wiped, and the NBT data is not, and the NBT data does not match the current "empty" state of the tool, until I click a slot in the workbench, at which point the NBT data is changed to "match" the current empty state of the tool.

**Comment by TheInfamousAlk** [comment]
Update: my capacitor backpack keeps deleting all power from it as well.

**Comment by jwols123** [comment]
this is also happening to me as well. I'm playing on the most recent version of the modpack "Cuboid Outpost" and the thing with items from the toolbox disappearing is happening with me as well. Why have they not looked into this more for the roughly 2 years it's been a problem?
Version: 5.0.7-143

**Comment by TheInfamousAlk** [comment]
Probably because we're already on ATM9. I had completely forgotten about this.
